site/app/libraries/Core.php

```diff
@@ -291,19 +262,7 @@ public function hasDBPerformanceWarning(): bool {
             return true;
         }
 
-        $queries = [];
-        if ($this->course_debug_stack !== null) {
-            foreach ($this->course_debug_stack->queries as $query) {
-                $queries[] = $query['sql'];
-            }
-        }
-        if ($this->submitty_debug_stack !== null) {
-            foreach ($this->submitty_debug_stack->queries as $query) {
-                $queries[] = $query['sql'];
-            }
-        }
-
-        return count($queries) !== count(array_unique($queries));
```

Comment:
We don't need this check anymore as the `hasDuplicateCheck()` that's run against the `$this->course_db` and `$this->submitty_db` cover what this was checking anyway, right?

Reply:
Yeah these were strictly for the ORM queries that were issued. So the `hasDuplicateCheck()` will now cover both.